pop-script.js

```diff
@@ -0,0 +1,69 @@
+const projectArray = [
+  {
+    id: 0,
+    title: 'Tonic',
+    desktopTitle: 'Tonic',
+    details: ['CANOPY', 'Back End Dev', 2015],
+    desktopDetails: ['CANOPY', 'Back End Dev', 2015],
+    image: 'img/tonic_pic.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
```

Comment:
- [ ] I think it would be better to add a scroll to your pop-up contents or ensure that they fit in a 100 viewport height of devices since some content is not visible

![image](https://user-images.githubusercontent.com/98453979/224149247-f0636a3e-46d4-49e3-bd08-4272611a493c.png)